### DISCUSSION TITLE
Change crew healing in outposts.

### DISCUSSION THREAD

**Discussion by tropmzai** [FakeFishGames/Barotrauma#8485] [Ideas and suggestions]
What? Crew healing in outposts? This ruins whole authenticity of the game. 

I mean, im not playing a fortnite or minecraft, where you can throw some thing under your legs and get full health. Im playing game about brutality and hardcore of seas on sattelite of Upiter. Here you need to properly think about all of your movements, because game mechanics unique and fully connected with such a concept. They are hard and a little annoying and this is the point, what makes game beautiful and authentic. So your health is the most problematic zone in all of "movements". You need to be worried, because whole expedition depends on health. Game literally force you to make some sort of strategy, based on amount of med and your current health status. For example, if you dont have enough morphie or bandages you wont rush raptor`s nest, because you will die! Now you can just avoid this, get to the closest outpost and instantly heal your team. This game function probably not declines matters of medicine, but really  lowering it down.

So, actually, yes, it makes sense. In outposts live medics. But you literally press a single button and your teammates being healed. 

What if make it like more cinematographic? Tritely make delay before teammate is completely healed. 10 or 20 seconds would be enough. And the cost might be equivalent to the cost of the med, what were spent to heal person. 

Or even make a big markup to heal in outpost hospitals, to motivate player heal crew by his own med. In that case, healing in outpost would be last resort.

Maybe it can be depend on reputation. Reputation bad = outpost wont heal your crew. Good - will.

Or outpost medics dont have that much meds to heal every boat, so they will help you only if you have near death teammate.

In other words, do something, what will make it more realistic function like other aspects of a game. And it will look like it was in game all the time.

**Comment by Ek-Videogames** [comment]
That would require devs properly balance afflictions and ship's medicine, and issues like accumulated 'micro'-burns not to mention reapers tax.

( Making medical beds capable of healing very small afflictions would at least solve the issue of having to waste medicine on menial injuries )

**Comment by leonard84** [comment]
> In other words, do something, what will make it more realistic function like other aspects of a game. 

I find it actually more realistic that you can heal on an outpost that is better equipped.

> So, actually, yes, it makes sense. In outposts live medics. But you literally press a single button and your teammates being healed.

Yeah, otherwise it is just your medic giving you Morphine and you are being healed. It is the same with the repair hull/components option, you click a button and everything is fixed - for a price.

> What if make it like more cinematographic? Tritely make delay before teammate is completely healed. 10 or 20 seconds would be enough. 

What difference does it make? I mean the one being healed is currently not even aware of it happening, so a delay wouldn't help in any way.

> For example, if you dont have enough morphie or bandages you wont rush raptor`s nest, because you will die! Now you can just avoid this, get to the closest outpost and instantly heal your team. 

If you enter an outpost your round is over so you won't enter the nest anyway. By the same logic you could say you just go to the outpost to buy meds.

**Comment by GenericGoose** [comment]
this is a nice quality of life feature to remove tiny injuries
the real issue with outpost medics is the awful ui that breaks in every possible way 
pressing literally any button has a 20% chance of completely breaking in a multitude of ways
